src/jdk.jdeps/share/classes/com/sun/tools/jdeprscan/Main.java

```diff
@@ -414,7 +414,7 @@ boolean processRelease(String release, Collection<String> classes) throws IOExce
                              .noneMatch(n -> n.equals(release))) {
                 return false;
             }
-            JavaFileManager fm = pp.getPlatform(release, "").getFileManager();
+            JavaFileManager fm = pp.getPlatformTrusted(release).getFileManager();
```

Comment:
Not sure if this change is necessary. I believe `release` is verified to be a valid platform name at this point, so even with the new check, it should still work. (And `getPlatformTrusted` could possibly be eliminated.) But maybe I am missing something?

Reply:
I wanted to avoid adding exception handling code here, since `PlatformNotSupported` is a checked exception.

Reply:
Ah, OK, I didn't realize that. Seems fine then!